src/main.js

```diff
@@ -19,22 +20,22 @@ const listFunctionsFiles = async function (
   srcFolder,
   { jsBundler = JS_BUNDLER_ZISI, jsExternalModules, jsIgnoredModules } = {},
 ) {
-  const [functionInfos, pluginsModulesPath] = await Promise.all([
-    getFunctionInfos(srcFolder),
-    getPluginsModulesPath(srcFolder),
-  ])
+  const paths = await listFunctionsDirectory(srcFolder)
+  const functions = await runtimes.getFunctionsFromPaths(paths)
+  const pluginsModulesPath = await getPluginsModulesPath(srcFolder)
```

Comment:
Should `listFunctionsDirectory()` and `runtimes.getFunctionsFromPaths()` be performed in parallel to `getPluginsModulesPath()`? (for performance)

Reply:
Good catch!

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/364/commits/493e24550351a8124874c9462f2a4e2513815a0e. Thanks for the suggestion!